dev/sbt-checkstyle

```diff
@@ -17,17 +17,13 @@
 # limitations under the License.
 #
 
+build_profiles=$1
+echo "Checking Spark style using SBT with these arguments: ${build_profiles}"
```

Comment:
I think it's more consistent to print this in `run-tests.py`. You could do, for instance,

```
print("[info] Checking Java style using SBT with these arguments ...
```